src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
+    ? people.filter(p => p.sex === 'm'
+      && people.some(m => m.name === p.mother))
```

Comment:
Same as before ([styleguide 8.3](https://mate-academy.github.io/style-guides/javascript.html#83-in-case-the-expression-spans-over-multiple-lines-wrap-it-in-parentheses-for-better-readability)).